packages/core/apps/core-apps-browser-internal/src/status/lib/load_status.ts

```diff
@@ -57,16 +58,10 @@ function formatMetrics({ metrics }: StatusResponse): Metric[] {
   }
 
   return [
-    {
-      name: i18n.translate('core.statusPage.metricsTiles.columns.heapTotalHeader', {
-        defaultMessage: 'Heap total',
-      }),
-      value: metrics.process.memory.heap.size_limit,
-      type: 'byte',
-    },
     {
       name: i18n.translate('core.statusPage.metricsTiles.columns.heapUsedHeader', {
-        defaultMessage: 'Heap used',
+        defaultMessage: 'Heap used out of {heapTotal}',
+        values: { heapTotal: numeral(metrics.process.memory.heap.size_limit).format('0.00 b') },
```

Comment:
To avoid restructuring the whole status page, I combined the two separate heap tiles into one that includes both `heapUsedInBytes` and `sizeLimit`.